source/Irrlicht/COpenGLDriver.cpp

```diff
@@ -2159,10 +2159,6 @@ bool COpenGLDriver::SAuxContext::setActiveTexture(uint32_t stage, core::smart_re
 	if (stage >= COpenGLExtensionHandler::MaxTextureUnits)
 		return false;
 
-
-    if (texture&&texture->getVirtualTextureType()==IRenderableVirtualTexture::EVTT_BUFFER_OBJECT&&!static_cast<COpenGLTextureBufferObject*>(texture.get())->rebindRevalidate())
```

Comment:
rebindRevalidate used to recreate the binding between the TBO object and a buffer that's changed on the OpenGL side, now if you detect a change you should just remove the TBO object , recreate it with the changes and then update the descriptor set (the descriptor set layout must be created with allow update after bind).

Reply:
@Crisspl ?

Reply:
resolved